presets/4.5/filters/aos_rc/aos_65mm_filters.txt

```diff
@@ -0,0 +1,37 @@
+#$ TITLE: AOS 65mm Filters
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: FILTERS
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: filter, filters, aos, chris, rosser, 65mm, tiny, whoop
+#$ AUTHOR: Chris Rosser
+#$ DESCRIPTION: Developed for a 65mm Tiny whoop. Based on a 300mAh 1S build with 23000KV Motors.
+#$ DESCRIPTION: NOTE this needs bidirectional Dshot support and RPM filtering active to use. DO NOT ATEMPT TO USE WITH OUT RPM FILTERING!
+#$ DESCRIPTION: Follow the usual process of hover testing and safely checking out your tune before using. USE AT YOUR OWN RISK.
+#$ DISCUSSION: https://www.aos-rc.com
+#$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
+#$ INCLUDE: presets/4.5/filters/defaults.txt
+
+# -- Gyro lowpass filters --
+set gyro_lpf1_dyn_min_hz = 0
+set gyro_lpf1_dyn_max_hz = 0
+set gyro_lpf1_static_hz = 0
+set gyro_lpf2_static_hz = 1000
+set simplified_gyro_filter = OFF
+
+# -- Gyro Dynamic Notches --
+set dyn_notch_count = 1
+set dyn_notch_q = 500
+set dyn_notch_min_hz = 150
+
+# -- RPM filtering --
+set dshot_bidir = ON
+set rpm_filter_harmonics = 1
```

Comment:
much better to use the filter dimming  rather than turning off  harmonics. the first harmonic doesn't do a lot with 3 blade props while the second is doing work. the second doesn't have a big impact on the latency unlike the first. Dimming is smarter and leans into improvements in Betaflight 4.5

Reply:
The RPMs on the motors are so high that the first notch is usually 400-500Hz. The first harmonic is at 1000Hz and the third is at 1500Hz. Better to save the CPU cycles as the first and second harmonic are at frequencies well attenuated by built in gyro lowpass.